src/components/Modal/index.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { Card } from "react-materialize";
+
+class Modal extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.closeModal = this.closeModal.bind(this);
+  }
+  closeModal() {
+    this.props.closeModal();
+  }
+  render() {
+    const { info } = this.props;
+    const style = ["modal"];
+    if (this.props.scrollbox) {
+      style.push("scrollbox");
+    }
+    return (
+      <div className={style.join(" ")}>
```

Comment:
maybe do this instead:
```
const { info, scrollbox } = this.props

<div className={`modal ${scrollbox ? "scrollbox" : ""}`}>
```
personal preference but either way works

Reply:
I'll try this. Definitely is less verbose